### DISCUSSION TITLE
Bed level changes after successful print

### DISCUSSION THREAD

**Discussion by jvcjbl** [mriscoc/Ender3V2S1#110] [Q&A]
I have an Ender 3 V2 with an upgraded bed. The bed is 5mm thing and guaranteed flat across the whole surface to be equal to or less than .10mm. I've seen WELL under this when using jyers firmware. Since switching to this firmware I feel that I'm having to level the bed constantly. Last night for example. I leveled the bed using the tramming wizard. 0.00 on all corners. Then ran the auto level and the bed was anywhere from -.05 to .09 which is way out of the specs I've seen. Just for kicks, I flashed back to jyers and the bed damn near zeros across the board which is what I've seen for days on end while using that firmware. So last night, after leveling the bed and seeing that weird spread, I printed a 3DBenchy and it was super clean. I was excited, thinking I finally got it dialed in with this firmware. Just for kicks I checked the bed with the taming wizard and both left sides were .09 and both right side were -0.13. What would cause this to change so drastically. I'm the only one awake during this time... I know I didn't touch the knobs. Nothing is loose. I have bushing on my bed and everything is square/plumb. All other firmware I've used, once I got the bed level it was kind of set it and forget it. Ideas? I'm tempted to switch back to jyers for a few days and see if it remains how it was prior.

**Comment by mriscoc** [comment]
This firmware uses by default higher speeds to move Z axis while doing the auto level. You can slow that speed in Control / Motion / Speed and also in Advanced / Probe settings / Z feed rate. I have a Ender 3v2 and my bed tramming and meshes are very consistent even at default speeds but some printers with mechanical/backlash issues would not be able to tolerate those speeds.

**Reply by jvcjbl** [reply]
Is there a way to turn off the heating time out?

**Reply by mriscoc** [reply]
Yes, commenting this line in Configuration_adv.h
```cpp
#define HOTEND_IDLE_TIMEOUT
```
But I don't recommend doing it, this time-out prevents the filament residues crystallizing and clog your hot end.

**Comment by GootyS** [comment]
I have been chasing my tail some with the ender S1 on Bed tramming on the 14/03//22 firmware., also seem to be constantly leveling the bed. Its a great firmware though no lies there.

**Reply by mriscoc** [reply]
Update to the latest version, it has fixed a z-offset bug.
